<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSurfaceMirrorModule</class>
 <widget class="qSlicerWidget" name="qSlicerSurfaceMirrorModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="qMRMLTreeView" name="MRMLTreeView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sceneModelType" stdset="0">
      <string>ModelHierarchy</string>
     </property>
     <property name="nodeTypes" stdset="0">
      <stringlist>
       <string>vtkMRMLModelHierarchyNode</string>
       <string>vtkMRMLModelNode</string>
      </stringlist>
     </property>
     <property name="renameMenuActionVisible" stdset="0">
      <bool>true</bool>
     </property>
     <property name="deleteMenuActionVisible" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="mirrorCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Clip the Model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout1">
        <item>
         <widget class="QLabel" name="modelLabel">
          <property name="text">
           <string>Select a Model to Mirror</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="mirrorNodeComboBox" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLModelNode</string>
           </stringlist>
          </property>
          <property name="showHidden" stdset="0">
           <bool>false</bool>
          </property>
          <property name="showChildNodeTypes" stdset="0">
           <bool>false</bool>
          </property>
          <property name="hideChildNodeTypes" stdset="0">
           <stringlist/>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="renameEnabled" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="annotationLabel">
          <property name="text">
           <string>Select a Fiducial List to Create Plane (Optional)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="annotationNodeComboBox" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLAnnotationHierarchyNode</string>
           </stringlist>
          </property>
          <property name="showHidden" stdset="0">
           <bool>false</bool>
          </property>
          <property name="showChildNodeTypes" stdset="0">
           <bool>false</bool>
          </property>
          <property name="hideChildNodeTypes" stdset="0">
           <stringlist/>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="renameEnabled" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="createButton">
          <property name="text">
           <string>Create Plane</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="annotationCreateButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Create Plane with Fiducials</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="hideButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Hide Plane</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mirrorButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Mirror</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="hintText">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#ff0000;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLTreeView.h</header>
   <slots>
    <slot>setSceneModel(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>MRMLTreeView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerSurfaceMirrorModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>mirrorNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerSurfaceMirrorModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLTreeView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerSurfaceMirrorModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>annotationNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
